gdcdictionary/schemas/pathology_detail.yaml

```diff
@@ -782,6 +784,16 @@ properties:
     type: number
     minimum: 0
 
+  tumor_level_prostate:
```

Comment:
noting we didn't make this an array of enums per the ticket.

Reply:
This seems to be resolved.